drivers/arm/ws2812_spi.c

```diff
@@ -1 +1,90 @@
-#error("NOT SUPPORTED")
\ No newline at end of file
+#include "quantum.h"
+#include "ws2812.h"
+
+/* Adapted from https://github.com/gamazeps/ws2812b-chibios-SPIDMA/ */
+
+#ifdef RGBW
+#    error("NOT SUPPORTED")
```

Comment:
I'm just curious, is there a specific reason for it not being supported, or just "it hasn't been supported yet, and may be added at a later date"?

Reply:
Mostly no hardware to test, but i would rather make the assumption it could be added in the future than blindly add it now.

My main use case right now is #7694, and maybe catching the extra duplication within #7507.

Reply:
I have plenty 072 PCBs coming that could be used to test. See 
 (https://github.com/Gondolindrim/Tsuki)[Tsuki], (https://github.com/Gondolindrim/ArcticPCB)[Arctic], (https://github.com/Gondolindrim/KeebsPCB)[Keebs]. I have also three more coming that use RGB and 072 but arent publicly released yet.

Reply:
This isn't a F072 issue, but rather a RGBW one. I have development boards for testing, but not representative RGBW LEDs right now.

Reply:
I have a Planck rev6, and a RGBW strip that I can test with.  

I actually bought the RGBW strip for exactly that purpose.

Reply:
Planck might need a bit of a bodge to use (and avoid) the SPI pins. Much easier on a ST dev board, or Proton C.

Reply:
Ugh, looking into this, it is going to be a lot more difficult to support RGBW.  So yeah, definitely shelved for now.